ilastik/shell/gui/startShellGui.py

```diff
@@ -73,7 +73,7 @@ def _applyStyleSheet(app):
     Apply application-wide style-sheet rules.
     """
     styleSheetPath = os.path.join( os.path.split(__file__)[0], 'ilastik-style.qss' )
-    with file( styleSheetPath, 'r' ) as f:
+    with open( styleSheetPath, 'r' ) as f:
```

Comment:
Nice catch.  From a quick grep, it looks like the following files need the same change (`file` -> `open`).  (You might have caught these alreadly; this is a note for me to check.)

-  `ilastik/clusterConfig.py`
- `lazyflow/utility/io_util/RESTfulVolume.py`
- `lazyflow/utility/io_util/RESTfulVolume.py`

(Weird thing is, I didn't even know that `file()` can substitute for `open()`, but apparently I wrote all these lines... Strange that I made this mistake without even knowing it was legal.)